subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -38,6 +38,12 @@ Some plugins will break with this new version of Gradle, for example because the
 
 === Potential breaking changes
 
+==== Version catalog only accepts a single TOML import file
+
+Only a single file will be accepted when using a `from` import method.
+We advise users with this use case to use a code-based version catalog.
```

Comment:
I would not highlight an alternative to handle this. I would instead clarify that this was possible only when passing multiple files in a single `from` call, so that most folks know they can ignore this point.